src/main.js

```diff
@@ -64,6 +65,65 @@ downloadButton.addEventListener("click", _ => {
     download(blob, "image.png", "image/png");
   });
 });
+
+
+function contrastTextColor(hex) {
```

Comment:
The current version has hardcoded coloured blocks and either white or black text.  
As we are moving the colours to json, rather than decide (in that json) to have white or black text, I wrote a little routing to work out which colour (black or white) was best.

Reply:
Would it be more useful/usable to reference these colours by their named defined in [the palette](https://design.theguardian.com/#colour-palette-4)? That is, `news` rather than `red`?

Reply:
I've discussed this with Ana.  The problem is (partly) that those names are big.  eg:
```
white
news bright (red)
brand blue (blue)
sport bright (bright blue #00b2ff)
highlight yellow (yellow)
grey
culture main (ochre)
lifestyle dark (purple)
opinion main (orange)
black
lifestyle bright (pink)
```

I thought it might be best to have a discussion about naming after this PR.